src/orange/surf/Plane.cc

```diff
@@ -0,0 +1,31 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/surf/Plane.cc
+//---------------------------------------------------------------------------//
+#include "Plane.hh"
+
+#include "PlaneAligned.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Promote from an axis-aligned plane.
+ */
+template<Axis T>
+Plane::Plane(PlaneAligned<T> const& other) noexcept
+    : normal_{other.calc_normal({0, 0, 0})}, d_{other.position()}
```

Comment:
Could you do `calc_normal(Real3{})` since the argument isn't actually used in PlaneAligned::calc_normal? Or maybe just have a `calc_normal()` that calls `calc_normal(Real3)`? At first glance this looked like a bug since `{0, 0, 0}` is only on the plane when `other.position == 0`.

Reply:
Good point, thanks. I'll add a `calc_normal()` method to the PlaneAligned that returns the normal, then add a `calc_normal(Real3)` to fulfill the necessary interface.